src/components/main/Archives.jsx

```diff
@@ -10,7 +10,7 @@ export default class Archives extends BaseComponent {
       this.props.archivesData.map(issue => (
         <div key={issue.issueNumber} className="archives__issue-item">
           <Link to={`/issue/${issue.issueNumber}`}>
-            <h1 className="archives__issue-item__issue-number">{`Issue ${issue.issueNumber}`}</h1>
```

Comment:
Remember to also change the name of the CSS class in the styles folder: https://github.com/thegazelle-ad/gazelle-server/blob/3474bce146b025980957ccae079694661011898d/src/styles/components/_archives.scss#L12